### ISSUE TITLE
artisan domain:add command issue

### ISSUE THREAD

**Issue by asifu5057** [gecche/laravel-multidomain#79]
Hello,

I am using Laravel 8.12 and gecche/laravel-multidomain 4.0

When I run command `php artisan domain:add mysite.com` it throw below error message

**cannot use a scalar value as an array at
vendor/gecche/laravel-multidomain/src/Foundation/Console/AddDomainCommand.php:135**

Please help how can I fix this issue?

Thanks,
Asif

**Comment by gecche** [comment]
Hi,

it seems that there is a problem with your `domain.php` config file: within that config array there must be present the `domains` entry and it must be an array.

It seems that you have a scalar value for that entry instead of an array

Let me know if this helps

Cheers

Giacomo

**Comment by asifu5057** [comment]
Thank you but currently `domain.php` file is empty, what should I add there? there is no instructions?

**Comment by asifu5057** [comment]
Yes I have ran that command.

**Comment by asifu5057** [comment]
which provider I should publish after running the command? can you please guide me?

**Comment by gecche** [comment]
Ok... I think that something happened when you ran that command, maybe you already had a `domain.php` file in your config dir.

You can try to rerun that command with the option `--force`: it should overwrite the current config file.

Alternatively, you get the right config file in the vendor folder:

`vendor/gecche/laravel-multidomain/src/config/domain.php`

simply copy it in the config dir.

Then I think it should work 

Giacomo

**Comment by asifu5057** [comment]
Yes there was some issue in running the command, I rerun and publish all files by pressing the `0`, all providers successfully published and now its working fine. Thank you @gecche
